examples/gallery/histograms/blockm.py

```diff
@@ -16,13 +16,13 @@
 
 # Set the region for the plot
 region = [130, 152.5, 32.5, 52.5]
-# Define spacing in x and y direction (150 by 150 minute blocks)
+# Define spacing in x and y direction (150x150 arc-minutes blocks)
```

Comment:
```suggestion
# Define spacing in x and y direction (150x150 arc-minute blocks)
```
In my opinion the plural is not correct here since the following word is already plural. Maybe a native speaker like @maxrjones or @willschlitzer may have a look on this.

Reply:
I think the correct grammar is "arc-minute blocks"

Reply:
Thanks @willschlitzer!
Changed it, see commit https://github.com/GenericMappingTools/pygmt/pull/2278/commits/fef0f1028561dcf8d297ec146029973a5e1a2d45.